scipy/optimize/tests/test__numdiff.py

```diff
@@ -182,10 +181,10 @@ def jac_zero_jacobian(self, x):
         ])
 
     def fun_non_numpy(self, x):
-        return math.exp(x)
+        return np.exp(x)
 
     def jac_non_numpy(self, x):
-        return math.exp(x)
+        return np.exp(x)
```

Comment:
I suspect the whole point of this test was to see how non-numpy functions behaved.

Perhaps then this should be `[math.exp(xi) for xi in x]`?

Reply:
Well, sometimes it's just called with a float. Worked around that.